app/src/main/AndroidManifest.xml

```diff
@@ -17,7 +36,8 @@
       android:networkSecurityConfig="@xml/network_security_config"
       android:supportsRtl="true"
       android:theme="@style/AppTheme">
-
+    <activity android:name=".webserver.ZimHostActivity">
```

Comment:
I would put newer activities at the bottom of the file near the other activities

Reply:
Fixed 0e2f585cd4dfa2ccfdce5c95000ca6913de2c468